active/ScrollingView/ScrollingView.md

```diff
@@ -0,0 +1,2584 @@
+
+The ScrollableHeight property returns the value Max(0, ZoomFactor x ExtentHeight - ViewportHeight).
+
+When overpanning horizontally at the left edge, HorizontalOffset returns a negative value.
+
+When overpanning horizontally at the right edge, HorizontalOffset returns a value larger than ScrollableWidth.
+
+The same comments apply to the vertical dimension.
+
+
+### ScrollingView.ZoomFrom method
+
+Asynchronously adds zooming inertia
+
+When the centerPoint parameter is null, the methods use the center of the viewport as the zoom center point.
+
+When an inertiaDecayRate is provided to ZoomFrom, it is applied during the duration of the zoom and then reverts back to the default 0.95.
+
+Changing the ZoomFactor does not affect the `ScrollingView`'s ViewportWidth, ViewportHeight, ExtentWidth, ExtentHeight properties. It does potentially affect the HorizontalOffset, VerticalOffset, ScrollableWidth and ScrollableHeight properties.
+
+For instance, starting with HorizontalOffset=10, VerticalOffset=20, ZoomFactor=1 and changing the ZoomFactor to 2 with a centerPoint at (0, 0) results in HorizontalOffset=20, VerticalOffset=40.
+
+The ScrollableWidth being defined as Max(0, ZoomFactor x ExtentWidth - ViewportWidth) is affected by the change too - it is increased.
+
+The ScrollableHeight being defined as Max(0, ZoomFactor x ExtentHeight - ViewportHeight) is affected by the change too - it is increased.
+
+
+### ScrollingView.AnchorRequested event
+
+Raised when the `ScrollingView` is about to select an anchor element. 
+
+The AnchorCandidates collection is populated with the elements it will pick from. The event handler can modify the content of this collection to influence the `ScrollingView`'s imminent selection, or it can set the Anchor to force the use of an element (the `ScrollingView`'s selection process is then skipped entirely).
+
+The `ScrollingView` uses its HorizontalAnchorRatio and VerticalAnchorRatio properties to select an anchor.
+
+
+### ScrollingView.HorizontalSnapPointsType (and VerticalSnapPointsType) property
+
+Gets or sets the type of snap points to use on the horizontal scrolling direction. Defaults to None.
+
+By default, because HorizontalSnapPointsType and VerticalSnapPointsType are set to None, no scrolling snap points are applied.
+
+When HorizontalSnapPointsType is set to Mandatory (the only other option available for now), the `ScrollingView` checks if its Content element implements the WUX <a href="https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.primitives.iscrollsnappointsinfo">IScrollSnapPointsInfo</a> interface.
+
+If the content does not implement that interface, no horizontal snap points are applied.
+
+If the content does implement that interface, the `ScrollingView` forwards the advertised snap points to its inner ScrollingPresenter control, using its IVector&lt;Microsoft.UI.Xaml.Controls.ScrollSnapPointBase&gt; HorizontalSnapPoints { get; } property.
```

Comment:
Sorry, this part of the doc was accidentally left here instead of being moved into the “Future Supported Scenarios” section.
There is no ScrollingView.HorizontalSnapPointsType/VerticalSnapPointsType property or support for IScrollSnapPointsInfo in the first release.

Reply:
Recommend: Move this to the Appendix